optimum/neuron/modeling_seq2seq.py

```diff
@@ -147,11 +182,29 @@ def _save_pretrained(
 
         for src_path, dst_path in zip(src_paths, dst_paths):
             dst_path.parent.mkdir(parents=True, exist_ok=True)
-            if src_path.is_file():
-                shutil.copyfile(src_path, dst_path)
+            shutil.copytree(src_path, dst_path, dirs_exist_ok=True)
 
         self.generation_config.save_pretrained(save_directory)
 
+    @staticmethod
+    def load_model(
+        encoder_path: Union[str, Path],
+        decoder_path: Union[str, Path],
+        tensor_parallel_size: int,
+    ):
+        if not tensor_parallel_size > 1:
```

Comment:
```suggestion
        if tensor_parallel_size == 1:
```
nit: It seems more clear to me